scripts/Muon/GUI/Common/corrections_tab_widget/background_corrections_model.py

```diff
@@ -43,6 +44,13 @@ def setup_functions(self):
         self.exp_decay = FunctionFactory.createFunction("ExpDecayMuon")
         self.exp_decay.setParameter("A", DEFAULT_A_VALUE)
 
+    def create_background_workspace(self) -> Workspace:
+        background = self.flat_background.getParameterValue(BACKGROUND_PARAM)
+        background_error = self.flat_background.getError(BACKGROUND_PARAM)
+        background_workspace = CreateSingleValuedWorkspace(DataValue=background, ErrorValue=background_error,
+                                                           StoreInADS=False)
```

Comment:
does it still work with project recovery and history if we dont put it in the ADS?